### PR TITLE
add fluentbit to  opensearch log shipping

### PR DESCRIPTION
**Pull request by jaketf**
both services are running still working on getting them to talk properly.

Note, revised the scope on this with @xpositivityx to remove SSL / custom users to a future PR. See https://github.com/massdriver-cloud/terraform-modules/issues/31

### CODE REVIEW

File: massdriver.yaml [hunk 1]
```diff
@@ -42,7 +42,75 @@ params:
     - cluster_networking
     - node_groups
     - core_services
+    - observability
   properties:
+    observability:
```
Comment (WillBeebe) [inline massdriver.yaml:47]: If we like this structure we can make this a type and ref it from each cloud's cluster.

**Review by WillBeebe — APPROVED** [review]
LGTM Jake

File: massdriver.yaml (hunk 1 above)
Comment (jaketf) [reply massdriver.yaml:47]: I like that idea. I'll back it out to a type once I confirm it's working as expected in staging

Comment (chrisghill) [reply massdriver.yaml:47]: I would advise not extracting this to a separate type.  EKS will have log options the others won't (Cloudwatch, managed Opensearch), GKE same (Stackdriver) and same with AKS.

We should use the same structure, but not force them to use the same type.

**Review by chrisghill — CHANGES_REQUESTED** [review]

**Pull request merged**